docs/ftldns/privacylevels.md

```diff
@@ -35,4 +35,8 @@ This setting disables
 - Query Log
 - Long-term database logging
 
+### Level 4 - disabled statistics
+Disables all statistics processing at all. Even the query counters will not be available.
+Make sure you have disabled logging to the file `/var/log/pihole.log` using `sudo pihole logging off`.
```

Comment:
Maybe phrase it as "Additionally, you can disable logging to the file ..." to note that this is an optional step. If there's something in the docs about how FTLDNS doesn't need the log file anyways, that can be linked.